apps/src/javalab/CommitDialogBody.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import i18n from '@cdo/javalab/locale';
+import color from '@cdo/apps/util/color';
+import CommitDialogFile from './CommitDialogFile';
+
+export const fileShape = {
+  name: PropTypes.string.isRequired,
+  commit: PropTypes.bool.isRequired,
+  hasConflictingName: PropTypes.bool.isRequired
+};
+
+const PADDING = 8;
```

Comment:
I could not for the life of me figure out how to import this constant from `CommitDialog`. It was `undefined` in `styles` object -- tried moving the `styles` declaration/assignment above the function declaration, but no luck. Some sort of hoisting thing I don't understand??

Reply:
I think you would need to do `export const PADDING` in `CommitDialog`, but it's a pretty minor copy/paste so I don't feel strongly about it needing to happen

Reply:
(git is being weird about my comment here so sorry if this is a duplicate): I think you need to do `export const PADDING = 8` and then you'll be able to import. It's a pretty minor duplicate so no worries if you can't make it work.

Reply:
I think I tried that, with an import statement like `import {PADDING} from ./CommitDialog` (does that look right) and just kept getting errors related to it being undefined. I guess I'll just leave it

Reply:
Just double checking here, looks like you have `const PADDING = 8` declared in `CommitDialog` but haven't exported it - does it work if you do `export const PADDING = 8`?

Reply:
It doesn't :(. Interestingly, when I did try that, I console logged the value of `styles` and `PADDING` in the component function, PADDING was set to 8 but the styles object had NaN in places where PADDING was used. ???